sdk/reservations/Azure.ResourceManager.Reservations/src/Custom/ReservationOrderResource.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.ResourceManager.Reservations.Models;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.Reservations
+{
+    /// <summary>
+    /// A Class representing a ReservationOrder along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="ReservationOrderResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetReservationOrderResource method.
+    /// Otherwise you can get one from its parent resource <see cref="TenantResource" /> using the GetReservationOrder method.
+    /// </summary>
```

Comment:
Could we remove the description here to make sure it will never override or append anything to the generated document which will change automatically along with the swagger?

Reply:
Removed